index.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
+
```

Comment:
Nah. Leave it. It's forward progress. I just wanted to make sure since I haven't studied strict mode (until now).